world/client.cpp

```diff
@@ -194,14 +194,19 @@ bool Client::CanTradeFVNoDropItem()
 void Client::SendEnterWorld(std::string name)
 {
 	std::string live_name {};
+
 	if (is_player_zoning) {
 		live_name = database.GetLiveChar(GetAccountID());
-		if(database.GetAccountIDByChar(live_name) != GetAccountID()) {
+		if (database.GetAccountIDByChar(live_name) != GetAccountID()) {
 			eqs->Close();
 			return;
 		} else {
 			LogInfo("Telling client to continue session");
 		}
+	} else {
+		if (RuleB(World, EnableAutoLogin)) {
+			live_name = AccountRepository::GetAutoLoginCharacterNameByAccountID(database, GetAccountID());
```

Comment:
For debugging purposes, it would be good to know if `live_name` ever ends up blank because we failed to find anything in the database. We assume happy path currently